web-client/src/client/lib.rs

```diff
@@ -856,6 +856,23 @@ impl Client {
         Ok(serde_wasm_bindgen::to_value(&txs)?.into())
     }
 
+    /// This function is used to tell the network to disconnect from every connected
+    /// peer and stop trying to connect to other peers.
+    #[wasm_bindgen(js_name = disconnectPeers)]
+    pub async fn disconnect_peers(&self) {
+        let network = self.inner.network();
+        network.disconnect(CloseReason::GoingOffline).await;
+    }
+
+    /// This function is used to tell the network to restart connecting to peers.
+    /// This is intended to tell the network to restart connection operations after
+    /// disconnect peers is called.
+    #[wasm_bindgen(js_name = restartConnecting)]
+    pub async fn restart_connecting(&self) {
+        let network = self.inner.network();
+        network.start_connecting().await;
+    }
```

Comment:
Is there any problem with calling this method while the client is connected as usual? Does it break anything?

I could imagine simply calling the methods `disconnect_network()` and `connect_network()`  (as we don't have a client.network subclass).